utils/s3/client.cc

```diff
@@ -28,6 +30,15 @@
 #include "db_clock.hh"
 #include "log.hh"
 
+template <>
+struct fmt::formatter<s3::tag> {
+    constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+    auto format(const s3::tag& tag, fmt::format_context& ctx) const {
+        return fmt::format_to(ctx.out(), "<Tag><Key>{}</Key><Value>{}</Value></Tag>",
```

Comment:
I heavily doubt that making formatter of whatever type produce XML string is a good idea

Reply:
hmm, kind of. the same idea came to my mind also. IMHO, to concatenate strings and to print using fmtlib have their pros and cons.. but since it's part of an implementation, and is not horribly ugly, so i went ahead.